traits/trait_types.py

```diff
@@ -149,11 +156,12 @@ def default_text_editor(trait, type=None):
     return TextEditor(auto_set=auto_set, enter_set=enter_set, evaluate=type)
 
 
+# -------------------------------------------------------------------------------
 # Generic validators
+# -------------------------------------------------------------------------------
 
 def _validate_int(value):
-    """
-    Convert an integer-like Python object to an int, or raise TypeError.
+    """ Convert an integer-like Python object to an int, or raise TypeError.
```

Comment:
This seems a bit subjective. I personally prefer the original style here, but I reluctantly agree that this style is more consistent with the rest of the file.

Reply:
Yes, I'm not entirely happy that there are a number of places where there wasn't a really good way to get a nice one-liner for the docstring.  I personally prefer the
```
""" Brief summary on one line.

More detailed information
"""
```
and
```
""" Brief summary for trivial thing.
"""
```
(or with the triple quotes on the same line if it will fit).